creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -695,7 +695,21 @@ const _issueUpdateReplicaSetOp = async (
         return response
       }
 
-      if (config.get('entityManagerReplicaSetEnabled')) {
+      // First try updateReplicaSet via URSM
+      // Fallback to EntityManager when relay errors
+      try {
+        await audiusLibs.contracts.UserReplicaSetManagerClient._updateReplicaSet(
```

Comment:
We still want to wait till the next release cycle to enable this, so maybe behind config flag